src/main/resources/ValidationMessages.properties

```diff
@@ -19,4 +19,5 @@ previous.balancesheet.not.equal=value_not_equal_to_previous_period_on_balance_sh
 unexpected.data=unexpected_data
 value.required=value_required
 empty.resource=empty_resource
-date.outside.currentPeriod=date_outside_current_period
\ No newline at end of file
+date.outside.currentPeriod=date_outside_current_period
+date.not.after.currentPeriod=date_not_after_current_period
```

Comment:
Shall we use `date.invalid` for now to maintain consistency with the small full `ApprovalValidator`?